projects/aws/eks-a-admin-image/build/validate-release-manifest-version.sh

```diff
@@ -12,8 +12,8 @@ n=0
 max_attempts=120
 delay=30
 while true; do
-    LATEST_EKSA_VERSION_S3=$(curl $EKSA_RELEASE_MANIFEST_S3_URL | yq e '.spec.latestVersion' -)
-    LATEST_EKSA_VERSION_CLOUDFRONT=$(curl $EKSA_RELEASE_MANIFEST_CDN_URL | yq e '.spec.latestVersion' -)
+    LATEST_EKSA_VERSION_S3=$(curl $EKSA_RELEASE_MANIFEST_S3_URL | yq e '.spec.releases[-1].version' -)
```

Comment:
This change is needed after we changed the logic to set the latest version in the release process. It potentially fixes the error
```console
Error: invalid version v0.17.5, no matching release found
```